jschart/index.js

```diff
@@ -2917,13 +2921,8 @@ function build_chart(chart) {
     .enter()
     .append("svg")
     .classed("svg", true)
-    .attr(
-      "width",
-      chart.dimensions.viewport_width +
-        chart.dimensions.margin.left +
-        chart.dimensions.margin.right
-    )
-    .attr("height", get_svg_height(chart));
+    .attr("viewBox", "0 0" + " " + (chart.dimensions.viewport_width + chart.dimensions.margin.left +
+      chart.dimensions.margin.right) + " " + (get_svg_height(chart) + 20) + "");
```

Comment:
What is the "+ 20" for?  That is just leaving extra space at the bottom.

Reply:
It is because, without extra space, the SVG is getting cropped due to the viewport.

Reply:
Something isn't the same with these changes.

An empty chart without the changes:

![image](https://user-images.githubusercontent.com/15911116/61793057-cf620600-ade3-11e9-8e77-5eb8851bebb8.png)

An empty chart with the changes:

![image](https://user-images.githubusercontent.com/15911116/61793018-b9544580-ade3-11e9-9096-d78efff77e05.png)

Reply:
Looks good.